modules/oc-by-example-content.adoc

```diff
@@ -1857,7 +1857,13 @@ Log in to a server
   oc login localhost:8443 --web --callback-port 8280
 
   # Log in to the external OIDC issuer through Auth Code + PKCE by starting a local server listening port 8080
-  oc login localhost:8443 --exec-plugin=oc-oidc --client-id=client-id --extra-scopes=email,profile --callback-port=8080
+  oc login localhost:8443 --exec-plugin=oc-oidc --client-id=client-id --extra-scopes=email,profile --callback-port=8080 --issuer-url=$ISSUER_URL
+
+  # Log in with an external OIDC
+  oc login --exec-plugin=oc-oidc --issuer-url=$ISSUER_URL --client-id=$CLI_CLIENT_ID --extra-scopes=email,profile --callback-port=8080
```

Comment:
Sorry if previous comment https://github.com/openshift/openshift-docs/pull/74184#discussion_r1554849352 was not clear. It meant: remove line 1860; line 1863 has same meaning of 1860. Below is suggestion of change:
```suggestion
  oc login --exec-plugin=oc-oidc --issuer-url=<issuer url> --client-id=<client id> --extra-scopes=email,profile --callback-port=8080
```